EcomLogin/src/App.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import './App.css';
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+
+import Login from "./components/login.component";
+import SignUp from "./components/signup.component";
+
+function App() {
+  return (<Router>
+    <div className="App">
+      <nav className="navbar navbar-expand-lg navbar-light fixed-top">
```

Comment:
is this for a navigation component? I'm currently working on it as recorded in the google doc